### ISSUE TITLE
FormError returned when a submission request is invalid is not public

### ISSUE THREAD

**Issue by agrosner** [bazaarvoice/bv-android-sdk#22]
```
class FormError {

    @SerializedName("FieldErrors")
    private Map<String, FieldError> fieldErrorMap;

    public Map<String, FieldError> getFieldErrorMap() {
        return fieldErrorMap;
    }
}
```

Unless we define a public static helper in same package, we cannot access this class. Also the same for `FieldError`
```
class FieldError {

    @SerializedName("Field")
    private String field;
    @SerializedName("Message")
    private String message;
    @SerializedName("Code")
    private String code;
}
```

**Comment by agrosner** [comment]
Same with the fields defined here: 
```
class ConversationsResponse {
    @SerializedName("HasErrors")
    private Boolean hasErrors;
    @SerializedName("Errors")
    private List<Error> errors;
    @SerializedName("FormErrors")
    private FormError formErrors;
    Boolean getHasErrors() {
        return hasErrors;
    }

    List<Error> getErrors() {
        return errors;
    }

    public FormError getFormErrors() {
        return formErrors;
    }
}
```
`hasErrors()` and `getErrors()`

**Comment by caseykulm** [comment]
Fixed in 4.1.4
